Jenkinsfile.cloud

```diff
@@ -101,4 +107,28 @@ node(env.NODE) {
             """
         }
     }
+
+    stage("Create AMI") {
+        docker.image(DOCKER_IMG).inside(DOCKER_ARGS) {
+            // not packaged yet, just build from src for now
+            sh "dnf install -y git golang"
+            sh "git clone https://github.com/coreos/mantle"
+            sh "mantle/build ore"
+
+            withCredentials([[$class: 'AmazonWebServicesCredentialsBinding',
+                              credentialsId: env['AWS_CREDENTIALS']]]) {
+                sh """
+                    # use a symlink so that our uploaded filename is unique
+                    ln -sf ${output}/cloud/latest/rhcos.vmdk rhcos-${commit}.vmdk
+                    mantle/bin/ore aws upload --region us-east-1 \
+                        --ami-name 'rhcos_dev_${commit[0..6]}' \
+                        --ami-description 'Red Hat CoreOS ${commit}' \
+                        --bucket 's3://${env.S3_PRIVATE_BUCKET}/rhcos/cloud' \
+                        --file rhcos-${commit}.vmdk \
+                        --name "rhcos_dev_${commit[0..6]}" \
```

Comment:
We don't need to change this right now, but I think down the line we should prefer version numbers over hashes, particularly truncated hashes.

Reply:
Right now, we're only publishing this metadata internally. We should also sync it to an S3 bucket or something?